README.md

```
@@ -37,20 +36,36 @@
 
 - **학교 포털 데이터 연동**을 통한 학점, 성적 등 학사 데이터 실시간 동기화
 - 학생별 학사 데이터와 졸업 요건을 자동 비교 분석하여 **부족한 학점 확인**
-<br/>
+  <br/>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: br

(MD033, no-inline-html)